docs/hub/datasets-cards.md

```diff
@@ -2,6 +2,27 @@
 
 Each dataset may be documented by the `README.md` file in the repository. This file is called a **dataset card**, and the Hugging Face Hub will render its contents on the dataset's main page. To inform users about how to responsibly use the data, it's a good idea to include information about any potential biases within the dataset. Generally, dataset cards help users understand the contents of the dataset and give context for how the dataset should be used.
 
-The [Dataset Card Creation Guide](https://github.com/huggingface/datasets/blob/main/templates/README_guide.md) gives an overview of what components are found in a dataset card. The process for creating a new dataset card is outlined in the [Create a dataset card](https://huggingface.co/docs/datasets/dataset_card) guide.
+You can also add dataset metadata to your card. The metadata describes important information about a dataset such as its license, language, and size. It also contains tags to help users discover a dataset on the Hub. Tags are defined in a YAML metadata section at the top of the `README.md` file. Supported tags include:
 
-Reading through existing dataset cards, such as the [ELI5 dataset card](https://github.com/huggingface/datasets/blob/main/datasets/eli5/README.md), is a great way to familiarize yourself with the common conventions. There is also an [interactive dataset card builder](https://huggingface.co/datasets/card-creator/) that can guide you through creating your card.
+```yaml
+annotations_creators: List[str]
+language_creators: List[str]
+language: List[str]
+language_details: Optional[str]
+license: Union[str, List[str]]
+license_details: str
+pretty_name: str
+size_categories: List[str]
+source_datasets: List[str]
+task_categories: List[str]
+task_ids: List[str]
+paperswithcode_id: str
+train-eval-index: List[Dict]
+configs: List[str]
+extra_gated_fields: Dict
+extra_gated_prompt: str
```

Comment:
to help users filter datasets by keywords, they can use `tags`:
```suggestion
extra_gated_prompt: str
tags: List[str]
```

Reply:
yes, can't believe i missed that one!

Reply:
I would actually put `tags` as one of the first fields